.teamcity/settings.kts

```diff
@@ -120,7 +120,7 @@ fun BuildSteps.maven(goal: String, args: String?) {
     this.maven {
         pomLocation = "common-custom-user-data-maven-extension/pom.xml"
         goals = goal
-        runnerArgs = "-Dgradle.enterprise.url=https://e.grdev.net" + (args?.let { " $it" } ?: "")
+        runnerArgs = "-Dgradle.enterprise.url=https://e.grdev.net -Dgradle.enterprise.storage.directory=%teamcity.build.tempDir%/.gradle-enterprise" + (args?.let { " $it" } ?: "")
```

Comment:
Aha! So `gradle.enterprise.storage.directory` is a configuration parameter that our Gradle Maven extension supports. And that directory basically stores the equivalent of what the GE Gradle plugin stores under `~/.gradle/enterprise` by default.

Reply:
Yep: https://docs.gradle.com/enterprise/maven-extension/#anatomy_of_the_gradle_enterprise_directory